.github/workflows/flask.yml

```
@@ -0,0 +1,33 @@
+name: flask-testsuite
+on:
+  push:
```

```suggestion
  push:
    branches:
      - master
```

This will keep us from getting double builds on PRs